lib/mixlib/install/backend/bintray.rb

```diff
@@ -0,0 +1,210 @@
+#
+# Author:: Patrick Wright (<patrick@chef.io>)
+# Copyright:: Copyright (c) 2016 Chef, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require "mixlib/install/backend/base"
+
+#
+# Add method to Array class to support
+# searching for substrings that match against
+# the items in the Array
+#
+class Array
+  def fuzzy_include?(search_value, regex_format = "%s")
+    inject(false) do |is_found, array_value|
+      is_found || !!(search_value =~ /#{regex_format % array_value}/)
+    end
+  end
+end
+
+module Mixlib
+  class Install
+    class Backend
+      class Bintray < Base
+        ENDPOINT = "https://bintray.com".freeze
+
+        API_VERSION = "v1".freeze
+
+        BINTRAY_ORGANIZATION = "chef".freeze
+
+        BINTRAY_USERNAME = "mixlib-install@chef".freeze
```

Comment:
It might be worthwhile to throw a comment above this username/password explaining that this is intentional and not an accidental leaking of credentials.